jep/226/README.adoc

```diff
@@ -122,12 +122,25 @@ For an external storage implementation, this creates a problem because the exter
 
 This proposal offers the following solution:
 
-* When a build/job is deleted from Jenkins, a subsequent update is performed to the fingerprint to reflect this change
 * An async periodic work is run to check if any fingerprint has no jobs/builds associated with it and delete it. (But, it would ensure that if there is a deletion blocking `FingerprintFacet`, then the fingerprint would not be deleted.)
 
-The above solution is provided to the user as an option, and can be toggled because it may be the case that storing data (i.e. fingerprints) might be cheaper than to do consistent updates in the database.
+New methods introduced inside the `FingerprintStorage` class:
+
+* `void setupPeriodicFingerprintCleanup(boolean toggle, @CheckForNull int recurrence_period)`
```

Comment:
recurrence_period should be long, also have users pass `TimeUnit` as a separate parameter.

Reply:
or even better, use `Duration`: https://github.com/a0x8o/kafka/blob/master/clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java#L65